tests/unit/commands/test_filter.py

```diff
@@ -31,28 +31,28 @@ def test_no_arg() -> None:
 
 
```

Comment:
Can you add a test that covers the case when the quotes around a filter expression are missing?

Reply:
Added two more tests with no quotes:
```
+def test_no_quotes_0() -> None:
+    line = 'filter obj'
+
+    with pytest.raises(sdb.CommandInvalidInputError):
+        invoke(MOCK_PROGRAM, [], line)
+
+
+def test_no_quotes_1() -> None:
+    line = 'filter obj == 1'
+
+    with pytest.raises(sdb.CommandArgumentsError):
+        invoke(MOCK_PROGRAM, [], line)
```
Let me know if you have anything more specific in mind :-)

Reply:
looks good!